Neos.ContentRepository/Classes/Domain/Model/Node.php

```diff
@@ -1731,7 +1731,11 @@ public function createVariantForContext($context)
          * @var $autoCreatedChildNode NodeInterface
          */
         foreach ($autoCreatedChildNodes as $autoCreatedChildNode) {
-            $autoCreatedChildNode->createVariantForContext($context);
+            $existingChildNode = $node->getNode($autoCreatedChildNode->getName());
+            if ($existingChildNode === null || !$existingChildNode->dimensionsAreMatchingTargetDimensionValues()) {
+                // only if needed, see https://github.com/neos/neos-development-collection/issues/782
```

Comment:
uhm do we really want such a reference in the code?

Reply:
Why not? Anyway, feel free to remove it.